packages/mds-audit/api.ts

```diff
@@ -549,9 +541,8 @@ function api(app: express.Express): express.Express {
                 version: res.locals.version,
                 ...audit,
                 provider_vehicle_id: device.vehicle_id,
-                provider_event_type: providerEvent[0]?.event_type,
-                provider_event_type_reason: providerEvent[0]?.event_type_reason,
-                provider_status: EVENT_STATUS_MAP[providerEvent[0]?.event_type as VEHICLE_EVENT],
+                provider_event_types: providerEvent[0]?.event_types,
```

Comment:
I see you are just following the previously established pattern, but if you were to use `const [provider_event] = await db.readEventsWithTelemetry({..., limit: 1})` you could get rid of all the `[0]` indexing.